presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/SuiteDescribe.java

```diff
@@ -84,12 +84,11 @@ public void run()
     public static class SuiteDescribeOptions
     {
         private static final String DEFAULT_VALUE = "(default: ${DEFAULT-VALUE})";
-        private static final String TARGET = "presto-product-tests/target";
 
         @Option(names = "--suite", paramLabel = "<suite>", description = "Name of the suite to describe", required = true)
         public String suite;
 
-        @Option(names = "--test-jar", paramLabel = "<jar>", description = "Path to test JAR " + DEFAULT_VALUE, defaultValue = TARGET + "/presto-product-tests-${project.version}-executable.jar")
+        @Option(names = "--test-jar", paramLabel = "<jar>", description = "Path to test JAR " + DEFAULT_VALUE, defaultValue = "${product.tests.module}/${product.tests.module}-${project.version}-executable.jar")
```

Comment:
The `$` needs to be escaped (the existing code is wrong too): https://picocli.info/#_escaping_variables